packages/player/src/components/controls/forward-replay-control.tsx

```diff
@@ -44,6 +44,8 @@ export const ForwardReplayControl: React.FC<ForwardReplayControlProps> =
             seekingTime: seekingTime > 0 ? seekingTime : 0,
           })
         }
+
+        videoService.send('END_SEEKING')
```

Comment:
this was the solution to a pretty tricky state bug where the seeking time would linger and get added to other operations 🐛